src/Mod/Spreadsheet/Gui/Command.cpp

```diff
@@ -507,8 +507,8 @@ CmdSpreadsheetAlignBottom::CmdSpreadsheetAlignBottom()
 {
     sAppModule = "Spreadsheet";
     sGroup = QT_TR_NOOP("Spreadsheet");
-    sMenuText = QT_TR_NOOP("Align &bottom");
-    sToolTipText = QT_TR_NOOP("Bottom-align contents of selected cells");
+    sMenuText = QT_TR_NOOP("Align &Bottom");

```

Comment:
```suggestion
    sMenuText = QT_TR_NOOP("&Bottom Align");
```
Pending Discord discussion